.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "stable"
+install: npm install
+services:
+  - redis-server
+script: npm test
```

Comment:
Same with this, it's not necessary to define it unless your tests do something other than `npm test`